modules/oadp-usecase-backup-odf.adoc

```diff
@@ -0,0 +1,257 @@
++
+.Example output
+[source,terminal]
+----
+# AWS_ACCESS_KEY_ID
+ebYR....xLNMc
+# AWS_SECRET_ACCESS_KEY
+YXf...+NaCkdyC3QPym
+----
+
+. Get the public URL for the S3 endpoint from the s3 route in the `openshift-storage` namespace by running the following command:
++
+[source,terminal]
+----
+$ oc get route s3 -n openshift-storage
+----
+
+. Create a `cloud-credentials` file with the object bucket credentials as shown in the following command. 
++
+[source,terminal]
+----
+[default]
+aws_access_key_id=<AWS_ACCESS_KEY_ID> # <1>
+aws_secret_access_key=<AWS_SECRET_ACCESS_KEY> # <2>
+----
+<1> Specify the AWS_ACCESS_KEY_ID.
+<2> Specify the AWS_SECRET_ACCESS_KEY.
+
+. Create the `cloud-credentials` secret with the `cloud-credentials` file content as shown in the following command.
++
+[source,terminal]
+----
+$ oc create secret generic \
+  cloud-credentials \
+  -n openshift-adp \
+  --from-file cloud=cloud-credentials
+----
+
+. Configure the DPA as shown in the following example:
++
+.Example DPA
+[source,yaml]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+  name: oadp-backup
+  namespace: openshift-adp
+spec:
+  configuration:
+    nodeAgent:
+      enable: true
+      uploaderType: kopia
+    velero:
+      defaultPlugins:
+        - aws
+        - openshift
+        - csi
+      defaultSnapshotMoveData: true # <1>
+  backupLocations:
+    - velero:
+        config:
+          profile: "default"
+          region: noobaa
+          s3Url: https://s3.openshift-storage.svc # <2>
+          s3ForcePathStyle: "true" 
+          insecureSkipTLSVerify: "true"
+        provider: aws
+        default: true
+        credential:
+          key: cloud
+          name:  cloud-credentials 
+        objectStorage:
+          bucket: <bucket_name> # <3>
+          prefix: oadp
+----
+<1> Set to true to use the {oadp-short} Data Mover to enable movement of Container Storage Interface (CSI) snapshots to a remote object storage.
+<2> The S3 URL of {odf-short} storage.
+<3> Specify the bucket name.
+
+. Create the DPA by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f <dpa_filename> # <1>
+----
+<1> Specify the DPA file name.
+
+. Verify that the DPA is created successfully by running the following command. In the example output, you can see the `status` object has `type` set to `Reconciled`. This means, the DPA is successfully created.
++
+[source,terminal]
+----
+$ oc get dpa -o yaml
+----
++
+.Example output
++
+[source,yaml]
+----
+apiVersion: v1
+items:
+- apiVersion: oadp.openshift.io/v1alpha1
+  kind: DataProtectionApplication
+  metadata:
+    namespace: openshift-adp
+    #...#
+  spec:
+    backupLocations:
+    - velero:
+        config:
+          #...#
+  status:
+    conditions:
+    - lastTransitionTime: "20....9:54:02Z"
+      message: Reconcile complete
+      reason: Complete
+      status: "True"
+      type: Reconciled
+kind: List
+metadata:
+  resourceVersion: ""
+----
+
```

Comment:
Anyone else think it would be a good idea to get customers in the habit of checking their BSL prior to running a backup?  The DPA can reconcile w/ bad creds, only the BSL will show if a customer is truly ready to execute a backup.
`oc get bsl -n openshift-adp`
```
oc get bsl
NAME           PHASE       LAST VALIDATED   AGE   DEFAULT
dpa-sample-1   Available   3s               15s   true
```

Reply:
Thank you. I will add this step.

Reply:
+1 +2 +3

Reply:
added this step.